activemq-transport/pom.xml

```diff
@@ -5,12 +5,12 @@
   <parent>
     <groupId>com.esri.geoevent.parent</groupId>
     <artifactId>activemq</artifactId>
-    <version>10.4.1</version>
+    <version>10.4.1-5.15.8</version>
```

Comment:
we prefer to keep the GES versions only, without the ActiveMQ version, for the transport (and adapter versions), since we are not going to release more than one transport version for `10.4.1`. Please revert these versions to `10.4.1`.

Reply:
Edited to 10.4.1 as you had it before, matching the GeoEvent SDK to be consistent with other components.